rag_api/main.py

```diff
@@ -51,18 +55,19 @@ async def startup_event():
     """Cache vector store at start-up."""
     try:
         initialize_vectorstore(document_location, "**/*.context")
+        global vector_store
+        print(f"Vector Store Loaded {vector_store}")
 
     except Exception as exc:
         print(str(exc))
 
 
 def initialize_vectorstore(doc_loc: str, doc_filter="**/*.*"):
     """Initialize and cache vector store interface."""
-
+    print("Initializing vector store...")
```

Comment:
Can this be moved to logger now instead of print?

Reply:
done